src/components/structures/UserSettings.js

```diff
@@ -328,12 +328,12 @@ module.exports = React.createClass({
             );
         }
         var notification_area;
-        if (!MatrixClientPeg.get().isGuest()) {
+        if (!MatrixClientPeg.get().isGuest() && this.state.threepids !== undefined) {
```

Comment:
surely we should still show the Notifications settings, even if the user has no threepids?